trieste/models/optimizer.py

```diff
@@ -154,23 +154,25 @@ def train_fn() -> None:
 
 @dataclass
 class KerasOptimizer:
-    """Optimizer wrapper for training neural network models implemented with Keras."""
+    """Optimizer wrapper for training models implemented with Keras."""
 
     optimizer: tf.keras.optimizers.Optimizer
     """ The underlying optimizer to use for training the model. """
 
-    loss: Union[
-        tf.keras.losses.Loss, Callable[[TensorType, tfp.distributions.Distribution], TensorType]
-    ] = None
-    """ Defines the loss function for training the network. """
-
     fit_args: dict[str, Any] = field(default_factory=lambda: {})
     """
-    The keyword arguments to pass to the `fit` method of a :class:`~tf.keras.Model` instance.
+    The keyword arguments to pass to the ``fit`` method of a :class:`~tf.keras.Model` instance.
     See https://keras.io/api/models/model_training_apis/#fit-method for a list of possible
     arguments in the dictionary.
     """
 
+    loss: Optional[
```

Comment:
You need to update the existing calls to KerasOptimizer in test.utils.models.keras.models and and test.unit.modles.keras.test_models (and maybe elsewhere too)